packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -267,3 +267,29 @@ class WalletCreditsMessage(RabbitMessageBase):
 
     def routing_key(self) -> str | None:
         return f"{self.wallet_id}"
+
+
+class CreditsLimit(IntEnum):
```

Comment:
MINOR: I would add suffix , i.e `CreditsLimitInt`

Reply:
Why an enum and not simply a int constant?

Reply:
Because I envision it as enum, this is just a plan under consideration, but my idea was as follows:
RUT will publish messages when wallet is under some reasonable defined credits limit (this PR implements just ZERO, but imagine we will add for example also 50 and 100) Then:
- director-v2 would subscribe to topic (which currently has structure <wallet-id>.<credits-limit-enum>): `*.0` --> because he is interested only when 0 is reached (maybe in the future -200 if we allow to go to negative, maybe there will be different action for 0 locking project for -200 killing services)
- webserver would subscribe to topic <specific-wallet-id>.* because on 100 he will maybe inform user that he/she is running out of credits, in 50 ones again more intrusive message will pop-up, on  0 ones again some action is taken by webserver/frontend

Idea under consideration:
- Payment Service (as o owner of auto-recharge thresholds) knows which wallet has auto-recharge turned on with what limit. Based on that payment service can subscribe to different topics based on his needs. For example, if for wallet_id 5 the limit chosen by the user is 40, then he can subscribe to topic 10.50, while if the limit of the user is chosen to be 500 then the payment service needs to subscribe to get all usage of that wallet. To make it a bit less intrusive we can create also some reasonable higher threshold such as 1000 (so payment service can subscribe if it is under this threshold and will receive less messages)

Let me know what you think. We can also discuss this in person. 
-

Reply:
Also btw: RabbitMQ's default topic-based routing doesn't support numerical comparison like "less than or equal to". Its topic routing is based on string patterns, with * (to match one word) and # (to match zero or multiple words).

Reply:
ok, discussed in person :+1: